iroha/benches/kura.rs

```diff
@@ -0,0 +1,58 @@
+#![allow(missing_docs, clippy::restriction)]
+
+use byte_unit::Byte;
+use criterion::{criterion_group, criterion_main, Criterion};
+use iroha::{kura::BlockStore, prelude::*, wsv::World};
+use iroha_crypto::KeyPair;
+use iroha_data_model::prelude::*;
+use tokio::{fs, runtime::Runtime};
+
+async fn measure_block_size_for_n_validators(n_validators: u32) {
+    let dir = tempfile::tempdir().unwrap();
+    let alice_xor_id = <Asset as Identifiable>::Id::from_names("xor", "test", "alice", "test");
+    let bob_xor_id = <Asset as Identifiable>::Id::from_names("xor", "test", "bob", "test");
+    let transfer = Instruction::Transfer(TransferBox {
+        source_id: IdBox::AssetId(alice_xor_id).into(),
+        object: Value::U32(10).into(),
+        destination_id: IdBox::AssetId(bob_xor_id).into(),
+    });
+    let keypair = KeyPair::generate().expect("Failed to generate KeyPair.");
+    let tx = Transaction::new(
+        vec![transfer],
+        <Account as Identifiable>::Id::new("alice", "wonderland"),
+        1000,
+    )
+    .sign(&keypair)
+    .expect("Failed to sign.");
+    let tx = VersionedAcceptedTransaction::from_transaction(tx, 4096)
+        .expect("Failed to accept Transaction.");
+    let mut block = PendingBlock::new(vec![tx])
+        .chain_first()
+        .validate(&WorldStateView::new(World::new()), &AllowAll.into());
+    for _ in 0..n_validators {
+        block = block
+            .sign(&KeyPair::generate().expect("Failed to generate KeyPair."))
+            .unwrap();
+    }
+    let block = block.commit();
+    let block_store = BlockStore::new(dir.path()).unwrap();
```

Comment:
Could it be implemented without storage, just using serialization?

Reply:
In general it could, but in this case we want to see the actual file size on disk.

Reply:
But serialization will give you some vector with exact byte count, no?